### PR TITLE
Feat/employee profile

### PR DESCRIPTION
**Pull request by Alabbey2**
employee-profile-screenshot

### CODE REVIEW

**Review by shresthapr — DISMISSED** [review]

File: client/src/Components/EmployeeProfile/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './EmployeeProfile.scss';
+import Header from '../Header/Header'
```
Comment (Chiranjibichapagain) [inline client/src/Components/EmployeeProfile/index.js]: Here you are building 2 components together. When you are working on the EmployeeeProfile component just work on that. You are building heard as well here which 1 is not a good practice and 2. there is no header component in our backlog, we have a navigation component which i am doing. so you don't have to worry about how the header will look like. just do employee profile here. :)

File: client/src/Components/EmployeeProfile/index.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import Button from '../Button/index'
+ 
+ 
+        const EmployeeProfile = () => {
+            return (
+                <div className="employee-profile">
+                    <header>
+                        <Header />
+                        <img className="employee-profile__image__small" src="/image/user.png" />
+
```
Comment (Chiranjibichapagain) [inline client/src/Components/EmployeeProfile/index.js]: exclude header from this component

File: client/src/Components/EmployeeProfile/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                    </header>
+                    
+                    <div className="employee-profile__main">
+                        <img className="employee-profile__image__big" src="/image/user.png" />
```
Comment (Chiranjibichapagain) [inline client/src/Components/EmployeeProfile/index.js]: the className convention when you use modifier is : "employee-profile__image employee-profile__image--big"  and  "employee-profile__image employee-profile__image--small"

File: client/src/Components/EmployeeProfile/index.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                            <p>Junior, Full-time/Part-time</p>
+
+                            <h3>About</h3>
+                            <p>
+                                is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing       
+                            </p>
+                            <p>React, Javascript, Redux, MongoDB</p>
+                            <div className="employee-profile__btn">
+                       
+                       <Button text="Edit Profile" modifier="md" />
```
Comment (Chiranjibichapagain) [inline client/src/Components/EmployeeProfile/index.js]: Every data you passed in should be dynamic and passed via props. You are hard coding the data here make dummy data and pass those data via props to the component from App.js to test the component.  Eg: "h2 Talent name h2 " not like this. get the talent name from props and use that name like: " h2 {talentName} h2 "

File: client/src/Components/EmployeeProfile/EmployeeProfile.scss [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+//mobile first
+.employee-profile{
+    width: 100%;
+    height: auto;
+}
+header {
+        display: flex;
+        margin-bottom: 2rem;
+        padding: 2rem;
+        .logo__main {
```
Comment (Chiranjibichapagain) [inline client/src/Components/EmployeeProfile/EmployeeProfile.scss]: We need to make our css style consistent. Please check the button component and try to align your css writing style with it. You can also check the better way of using modifiers there.

File: client/src/Components/EmployeeProfile/index.js (hunk 3 above)
Comment (Chiranjibichapagain) [reply client/src/Components/EmployeeProfile/index.js]: In fact, you don't need a modifier at all for this component. as navigation component will take care of the small profile image. you just need to create the image for the talent profile. And try with the real image not SVG.

**Review by Chiranjibichapagain — CHANGES_REQUESTED** [review]

**Comment by Alabbey2** [comment]
new_employee-profile-update

File: client/src/Components/EmployeeProfile/index.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+                            <h3>{title}</h3>
+                            <p>{position}</p>
+
+                            <h3>{about}</h3>
+                            <p>{description}</p>
+                            <p>{skills}</p>
+                            <div className="employee-profile__btn">
+                       
+                       <Button text="Edit Profile" modifier="md" />
+                    </div>
```
Comment (Chiranjibichapagain) [inline client/src/Components/EmployeeProfile/index.js]: Please review the SCSS reviewing style

**Review by Chiranjibichapagain — CHANGES_REQUESTED** [review]

**Comment by Alabbey2** [comment]
style1

**Comment by Alabbey2** [comment]
update

**Review by Chiranjibichapagain — DISMISSED** [review]

**Pull request merged**